examples/simple/main.go

```diff
@@ -39,9 +54,10 @@ func main() {
 			c.Enabled = true
 		},
 		Video: func(c *mediadevices.MediaTrackConstraints) {
-			c.Codec = webrtc.H264
+			c.Codec = videoCodecName
+			c.FrameFormat = frame.FormatI420 // most of the encoder accepts I420
 			c.Enabled = true
-			c.Width = 800
+			c.Width = 640
```

Comment:
I think most of the camera supports 640x480. Mine doesn't support 800x480...

Reply:
I see. I guess my camera is the weird one, lol